### ISSUE TITLE
Bug: Attribute Wrapping breaks Interpolated Tags

### ISSUE THREAD

**Issue by detroitenglish** [prettier/plugin-pug#149]
## Info

| Tool     | Version       |
| -------- | ------------- |
| Plugin   | v1.x.x        |
| Prettier | v2.x.x        |
| Node     | v12.x.x        |
| OS       | linux |

## Input

```pug

 strong #[a(href=link, target="_blank", rel="noopener")=link]
```
`.prettierrc`
```json
{
  "arrowParens": "avoid",
  "bracketSpacing": true,
  "endOfLine": "lf",
  "printWidth": 80,
  "pugAttributeSeparator": "always",
  "pugClosingBracketPosition": "new-line",
  "pugSingleQuote": false,
  "pugSortAttributes": "asc",
  "pugWrapAttributesPattern": "0",
  "pugWrapAttributesThreshold": 1,
  "semi": false,
  "singleQuote": true,
  "trailingComma": "es5"
}

```

## Output or Error
```pug
strong #[a(
  href=link,
  rel="noopener",
  target="_blank"
)= link]
```
```bash
  > 103|                 strong #[a(
------------------------------------^
    104|                   href=link,
    105|                   rel="noopener",
    106|                   target="_blank"

The end of the string reached with no closing bracket ) found.
```

## Expected Output

```pug

 strong #[a(href=link, target="_blank", rel="noopener")=link]
```

**Comment by Shinigami92** [comment]
I'm sorry to inform you that I will probably not have time until the weekend to keep working on it

**Comment by detroitenglish** [comment]
@Shinigami92 No stress! Thank you for all your hard work on this project 🙇‍♂️

**Comment by Shinigami92** [comment]
I've now fixed the bug because it corrupts the working pug code
But in the long run I think we should create an issue in the pug repo to support this wrapping within the pug code interpolation
In my opinion, pug should handle these types of wrapped attributes

**Comment by Shinigami92** [comment]
This has now been fixed in release `1.10.1`
Thank you for using my plugin and for being part of the pug community 🤗

**Comment by detroitenglish** [comment]
@Shinigami92 Thanks!
